src/main/resources/hudson/security/LDAPBindSecurityRealm.groovy

```diff
@@ -65,10 +65,11 @@ bindAuthenticator(BindAuthenticator2,initialDirContextFactory) {
     userSearch = ldapUserSearch;
 }
 
-authoritiesPopulator(AuthoritiesPopulatorImpl, initialDirContextFactory, instance.groupSearchBase) {
+authoritiesPopulator(AuthoritiesPopulatorImpl, initialDirContextFactory, instance.groupSearchBase, instance.groupMembershipStrategy) {
     // see DefaultLdapAuthoritiesPopulator for other possible configurations
     searchSubtree = true;
     groupSearchFilter = "(| (member={0}) (uniqueMember={0}) (memberUid={1}))";
+    ldapSearch = ldapUserSearch;
```

Comment:
Do I understand it correctly that this file can also be overridden by the user? See https://wiki.jenkins-ci.org/display/JENKINS/LDAP+Plugin#LDAPPlugin-Troubleshooting.

In that case, does this break if users have overridden this file without the additional constructor argument?